index.js

```diff
@@ -43,10 +43,13 @@ module.exports = {
 
     '@typescript-eslint/brace-style': ['error', '1tbs', { allowSingleLine: true }],
     '@typescript-eslint/comma-dangle': ['error', 'never'],
+    '@typescript-eslint/consistent-type-definitions': ['off'],
```

Comment:
```suggestion
    '@typescript-eslint/consistent-type-definitions': 'off',
```
If it's just `off`, maybe it doesn't have to be an array like others.

Reply:
> If it's just off, maybe it doesn't have to be an array like others.

Fixed: 797f8a0